communitiesuk/digital-form-builder-adapter
# FS-4038: Privacy, Cookies, Accessibility Statement and Contact Us Welsh translations

https://dluhcdigital.atlassian.net/browse/FS-2932

### Change description
- Privacy, Cookies, Accessibility Statement and Contact Us Welsh translations added



### How to test
- Go to apply, open a form in any fund round, notice if the Privacy, Cookies, Accessibility Statement and Contact Us Welsh translations are being shown at the webpage footer

### Screenshots of UI changes (if applicable)

<img width="1278" alt="Screenshot 2024-10-04 at 10 25 10" src="https://github.com/user-attachments/assets/48f02617-c5c8-449c-acc6-fc899aa5be4c">
